.vscode/launch.json

```diff
@@ -171,6 +171,11 @@
             "args": ["run", "${relativeFile}"],
             "smartStep": true,
             "console": "integratedTerminal",
+            "sourceMaps": true,
+            "outFiles": [
+                "${workspaceFolder}/packages/vscode-uri/lib/**/*.js",
```

Comment:
We don't have a package called `packages/vscode-uri` in our repo.

Reply:
Damn, True. It's supposed to be `node_modules` 🙈, but I think I'll just drop that line.

Reply:
I kept the line, and made it work.
I however dropped the line mentioning `langium`, as it doesn't bring any benefit.

That's most likely due to Vitest/Vite (executed via launch config) that one the fly translates the executed test module implemented in TS and all transitively referenced ones. However, it messes up the sourcemaps, or I wasn't able to configure it correctly.
Running the test via the Vitest VSCode extension works nicely in this regard 😐